stdlib/src/builtin/int.mojo

```diff
@@ -1212,6 +1213,37 @@ struct Int(
 
         writer.write(self)
 
+    @staticmethod
+    fn from_bytes[
+        D: DType, big_endian: Bool = False
```

Comment:
As mentioned above, I'm still opposed to adding a `DType` parameter to any `from_bytes`/`to_bytes` method for `Int`. We could either split these out into a new PR to nail down that API, or we can remove them.

Reply:
@lsh I agree. As it's also in all the SIMD scalar types, we don't need the `DType` parameter here. Good suggestion.

Fixed here: https://github.com/msaelices/mojo/commit/98c29fcdd490866e5e1f2ca4a85e201f6804bc85